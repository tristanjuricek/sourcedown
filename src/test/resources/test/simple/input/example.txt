# I'm a typical documentation file

Why not just allow normal markdown files too?