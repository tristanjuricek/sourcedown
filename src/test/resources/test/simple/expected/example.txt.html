<!DOCTYPE html>
<html lang="en">
<head>
  <title>example.txt</title>
  <meta http-equiv="content-type" content="text/html; charset=UTF-8">
  <link rel="stylesheet" href="./res/style.css"></style>
  <script type="text/javascript" 
    src="./res/jquery.js"></script>
</head>
<body>
  <div id="container">
    <h1 class="title">example.txt&nbsp;<span class="change-link">Change</span></h1>
    <ul id="nav-list" class="hidden">
                    
                <li><a href="./dir/example.scala.html">dir/example.scala</a></li>
                                
                <li><a href="./example.java.html">example.java</a></li>
                                
                <li class="active">example.txt</li>
                        </ul>
    <table><tbody><tr><td></td><td><pre class="brush: txt"># I'm a typical documentation file

Why not just allow normal markdown files too?</pre></td></tr></tbody></table>
  </div>
<script type="text/javascript">
$(function() {
    var $navList = $("#nav-list"),
        $change = $(".title .change-link");
    
    $change.click(function() { $navList.toggleClass("hidden") });
});
</script>
</body>
</html>
